v1.0/update.md

```diff
@@ -8,16 +8,399 @@ The `UPDATE` [statement](sql-statements.html) updates rows in a table.
 
 <div id="toc"></div>
 
-## Synopsis
+## Required Privileges 
 
-{% include sql/{{ page.version.version }}/diagrams/update.html %}
+The user must have the `SELECT` and `UPDATE` [privileges](privileges.html) on the table.
 
-## Required Privileges
+## Synopsis
 
-The user must have the `SELECT` and `UPDATE` [privileges](privileges.html) on the table.
+<div>{% include sql/v1.0/diagrams/update.html %}</div>
 
 ## Parameters
 
-| Parameter | Description |
-|-----------|-------------|
-|  |  |
+Parameter | Description
+----------|------------
+`table_name` | The name of the table that contains the rows you want to update.
+`AS name` | An alias for the table name. When an alias is provided, it completely hides the actual table name.
+`column_name` | The name of the column whose values you want to update. To update the values of multiple columns, use a comma-separated list of column names.
+`a_expr` | The new value you want to use, the [aggregate function](functions-and-operators.html#aggregate-functions) you want to perform, or the [value expression](sql-expressions.html) you want to use.
+`DEFAULT` | To fill columns with their [default values](default-value.html), use `DEFAULT VALUES` in place of `a_expr`. To fill a specific column with its default value, leave the value out of the `a_expr` or use `DEFAULT` at the appropriate position. 
+`column_name_list` | A comma-separated list of column names, in parentheses.
+`select_with_parens` | A comma-separated list of values or [value expressions](sql-expressions.html), in parentheses. To update values of multiple rows, use a comma-separated list of parentheses. <br><br>Each value must match the [data type](data-types.html) of its column. Also, if column names are listed (`qualified_name_list`), values must be in corresponding order; otherwise, they must follow the declared order of the columns in the table.
+`WHERE a_expr`| `a_expr` must be an expression that returns Boolean values using columns (e.g. `<column> = <value>`). Update rows that return `TRUE`.<br><br>__Without a `WHERE` clause in your statement, `UPDATE` updates all rows from the table.
```

Comment:
Yes, that's an error that creeped in when I was debugging the formatting issue. Fixed in the document.